ui/widgets/menu.js

```diff
@@ -142,7 +144,8 @@ return $.widget( "ui.menu", {
 			},
 			blur: function( event ) {
 				this._delay( function() {
-					if ( !$.contains( this.element[ 0 ], $.ui.safeActiveElement( this.document[ 0 ] ) ) ) {
+					if ( !$.contains( this.element[ 0 ],
```

Comment:
Let's move this contains call out of the conditional and store it in a variable, then put the variable in the conditional. It'll allow much nicer linebreaks.